index.js

```diff
@@ -1,32 +1,52 @@
+// ELEMENT_IDS
```

Comment:
If comment has the same value as line below, so I am not sure if there is a sense to keep such comment

Reply:
Solved